be/src/olap/cumulative_compaction_time_series_policy.cpp

```diff
@@ -229,6 +242,14 @@ int TimeSeriesCumulativeCompactionPolicy::pick_input_rowsets(
         return transient_size;
     }
 
+    auto consecutive_empty_rowsets = tablet->collect_consecutive_empty_rowsets();
```

Comment:
maybe we should treat it as condition 4 and put it in the last

Reply:
done